repos/system_upgrade/common/actors/cloud/checkhybridimage/actor.py

```diff
@@ -8,14 +8,9 @@
 class CheckHybridImage(Actor):
     """
     Check if the system is using Azure hybrid image.
-
-    These images have a default relative symlink to EFI
-    partition even when booted using BIOS and in such cases
-    GRUB is not able find "grubenv" to get the kernel cmdline
-    options and fails to boot after upgrade`.
     """
 
-    name = 'checkhybridimage'
+    name = 'check_hybrid_image'
     consumes = (InstalledRPM, FirmwareFacts)
     produces = (HybridImage, Report)
     tags = (ChecksPhaseTag, IPUWorkflowTag)
```

Comment:
the actor does not produce report anymore. and seems it could be moved to the FactsPhase

Reply:
discussed in chat, but the actor should renamed to scanner_hybrid_image as it's scanner now.